src/pages/groups/Group.js

```diff
@@ -98,6 +98,7 @@ export const GroupDetails = ajaxCaller(class GroupDetails extends Component {
           adminLabel: 'admin',
           userLabel: 'member',
           title: 'Add user to Terra Group',
+          checkRegistration: true,
```

Comment:
TOL: Having this override flag is fine for now, but we should keep an eye on this code path and decide if/when it's diverged enough to be split into two separate components instead.

Reply:
I agree. I feel like adding a user should be consistent throughout the application. At some point we may be able to remove the flag altogether